compile.sh

```diff
@@ -3,12 +3,10 @@
 set -e
 
 if [ `uname -p` = "x86_64" ]; then
-	echo "nwnx2 does not build on x64."
-	echo "Try running this in a 32bit chroot."
-	exit 1
+    export CC="gcc -m32" 
```

Comment:
This looks like an unrelated change

Reply:
hmm.. can I exclude this change from the PR?  or does it have to be reverted in my repo and re-committed?